HomeWork/src/main/java/exception/ArrayValueCalculator.java

```diff
@@ -0,0 +1,55 @@
+package exception;
+
+import string.StringLevelOne;
+
+import java.util.Arrays;
+import java.util.zip.DataFormatException;
+
+public class ArrayValueCalculator {
+
+    int results = 0;
+    String[][] str = new String[4][4];
+
+    public void arrayInitialization(String[][] strings1) {
+        for (int i = 0; i < strings1.length; i++) {
+            for (int j = 0; j < strings1[i].length; j++) {
+                strings1[i][j] = "5";
```

Comment:
You can make it more interesting by replacing `"5"` with randomly generated values, like this `ThreadLocalRandom.current().nextInt(5);` - it'll generate random values from 0 to 5